### DISCUSSION TITLE
Tailwind CSS does not apply with Dynamic import React component

### DISCUSSION THREAD

**Discussion by oukunan** [tailwindlabs/tailwindcss#12200] [Help]
Hello Tailwind Community 👋🏼

I attempted to configure NextJS with Tailwind CSS by doing the following:  https://tailwindcss.com/docs/guides/nextjs
Everything is good, but when I attempt to dynamically import a component,
```typescript
const Example: dynamic(() => import("@/components/Example")); 
...
<Example />
```
Look like CSS is not applied on my Example component

Can someone provide me with any suggestions? 
Do I require additional configuration? 
Any advice on what to do?

Thank you

**Comment by cojoclaudiu** [comment]
Same problem, did you find any solution?
